content/scout/env-vars.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Environment variables used by Docker Scout
+linkTitle: Docker Scout environment variables
+description: Configure how the behavior of Docker Scout CLI commands using these environment variables
+keywords: scout, supply chain, cli, env, variables, configure
+---
+
+The following environment variables are available to configure the Docker Scout
+CLI commands, and the corresponding `docker/scout-cli` container image:
+
+| Name                                    | Format  | Default | Description                                                                                 |
+| :-------------------------------------- | ------- | ------- | :------------------------------------------------------------------------------------------ |
+| DOCKER_SCOUT_CACHE_FORMAT               | String  | String  | Format of the local image cache; can be `oci` or `tar` (default: `oci`)                     |
+| DOCKER_SCOUT_CACHE_DIR                  | String  | String  | Directory where the local SBOM cache is stored (default: `$HOME/.docker/scout`)             |
+| DOCKER_SCOUT_NO_CACHE                   | Boolean | Boolean | When set to `true`, disables the use of local SBOM cache                                    |
+| DOCKER_SCOUT_OFFLINE                    | Boolean | Boolean | Use [offline mode](#offline-mode) when indexing SBOM                                        |
```

Comment:
Looks like the `Default` column is incorrect (doesn't show a value, but shows same as `Format`).